libcircularimage/circleImage.ets

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2022 Application Library Engineering Group.

```

Comment:
in libcircularimage folder, ets files are duplicated, pls remove the files outside src folder